scenes/SceneFilesFolder.js

```diff
@@ -15,9 +21,138 @@ import EmptyState from "~/components/core/EmptyState";
 
 const POLLING_INTERVAL = 10000;
 
+const STYLES_FILTERS_CONTAINER = css`
+  height: 60px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: flex-start;
+`;
+
+const STYLES_TOOLTIP_ANCHOR = css`
+  border: 1px solid #f2f2f2;
```

Comment:
Leave yourself a TODO here so you can add that to constants when we unify this into some design system